test/e2e/testing-manifests/statefulset/nginx/statefulset.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     spec:
       containers:
       - name: nginx
-        image: nginx:1.15-alpine
```

Comment:
Do we know where we load this yaml and make the modification there? using `imageutils.GetE2EImage` in the code in addition to modifying yaml here? ( the `GetE2EImage` allows folks to inject alternate repositories )

Reply:
In `test/e2e/framework/manifest/manifest.go`, StatefulSetFromManifest will use it. I changed that.